.env

```diff
@@ -10,6 +10,8 @@ S3_URL_EXTERNAL=http://localhost:9000
 S3_BUCKET=alfalfa
 REDIS_HOST=redis
 JOB_QUEUE_URL=http://goaws:4100/queue/local-queue1
+JOB_QUEUE_URL_1=http://goaws:4100/queue/local-queue1
+JOB_QUEUE_URL_2=http://goaws:4100/queue/local-queue2
```

Comment:
Is there a reason to not make the local queue names more descriptive?

Reply:
yes, i think we'll want to make the local-queues more descriptive such as local-fmu, local-openstudio...